src/applications/vaos/utils/selectors.js

```diff
@@ -544,45 +544,74 @@ export function selectActiveExpressCareFacility(state, nowMoment) {
   };
 }
 
-/*
- * Gets the formatted days and hours string of the first support facility.
- * 
- * Note: we're picking the first facility, there could be more than one
+function getFormattedTime(time) {
+  return moment(`${moment().format('YYYY-MM-DD')}T${time}:00`).format('h:mm a');
+}
+
+function getWindowString(window, timezoneAbbreviation, isToday) {
+  return `${isToday ? 'today' : titleCase(window.day)} from ${getFormattedTime(
+    window.startTime,
+  )} to ${getFormattedTime(window.endTime)} ${timezoneAbbreviation}`;
+}
+
+/**
+ * Returns next schedulable window.  If today is schedulable and current time is before window,
+ * return today's window.  Otherwise, return the next schedulable day's window
  */
-export function selectExpressCareHours(state) {
+export function selectNextAvailableExpressCareWindowString(state, nowMoment) {
   if (!state.expressCare.supportedFacilities?.length) {
     return null;
   }
 
   const facility = state.expressCare.supportedFacilities[0];
   const siteId = facility.facilityId.substring(0, 3);
+  const { timezone } = getTimezoneBySystemId(siteId);
   const timezoneAbbreviation = getTimezoneAbbrBySystemId(siteId);
-
-  // Trying to group days with the same times together
-  const windows = new Map();
-  facility.days.forEach(day => {
-    const key = `${day.startTime}_${day.endTime}`;
-    if (windows.has(key)) {
-      windows.get(key).push(day.day);
+  const nowFacilityTime = nowMoment.clone().tz(timezone);
+  const dayOfWeek = nowFacilityTime.format('dddd').toUpperCase();
+  const todayDayOfWeekIndex = Number(nowFacilityTime.format('d'));
+  const todaysWindow = facility.days.find(d => d.day === dayOfWeek);
+
+  // Sort schedulable days starting from today so we can easily find the next
+  // day if needed
+  const daysSortedStartingToday = [
+    todaysWindow,
+    ...facility.days.filter(day => day.dayOfWeekIndex > todayDayOfWeekIndex),
+    ...facility.days.filter(day => day.dayOfWeekIndex < todayDayOfWeekIndex),
+  ].filter(day => !!day);
```

Comment:
When would `day` be falsy?

Reply:
`todaysWindow` could be falsy, but we don't actually need it to be in this array so I removed it and the filter.